tools/codegen/gofilegen/codetemplate/template.go

```diff
@@ -27,11 +27,11 @@ type ResourceFileInputs struct {
 }
 
 type APIOperations struct {
+	Create        *Operation
+	Read          *Operation
+	Update        *Operation
```

Comment:
Similar to change in the codespec/model.go, dont believe we currently have a case where these operations are not defined